core/payment-driver/zksync/src/zksync/faucet.rs

```diff
@@ -25,8 +25,8 @@ lazy_static! {
     static ref MAX_WAIT: Duration = Duration::minutes(1);
 }
 
-pub async fn request_ngnt(address: &str) -> Result<(), GenericError> {
-    let balance = account_balance(address).await?;
+pub async fn request_ngnt(address: &str, network: Network) -> Result<(), GenericError> {
```

Comment:
We won't request funds on mainnet, so I think this parameter is not needed. (It will be only if we introduce more testnets, I believe).

Reply:
indeed, but the get_provider and get_balance functions need this argument.

I think its also neater to already include network here for when test-nets get added

Reply:
Alright.